### PR TITLE
container_registry: Container Analysis v1

### PR DESCRIPTION
**Pull request by DanSanche**
Updated Container analysis samples to use new v1 libraries

### CODE REVIEW

File: container_registry/container_analysis/create_occurrence.go [hunk 1]
```diff
@@ -41,16 +40,27 @@ func createOccurrence(resourceURL, noteID, occProjectID, noteProjectID string) (
...
+							AffectedPackage: "your-package-here",
+							MinAffectedVersion: &grafeaspb.Version{
+								Kind: grafeaspb.Version_MINIMUM,
+							},
+							FixedVersion: &grafeaspb.Version{
+								Kind: grafeaspb.Version_MAXIMUM,
+							},
+						},
+					},
+				},
```
Comment (DanSanche) [inline container_registry/container_analysis/create_occurrence.go]: FYI @Katee : It looks like the minimal Note and Occurrence has got a lot more complicated in v1 compared to beta1. Just wanted to make sure that was intentional (and let me know if this looks like a good sample, or if you have another idea in mind)

File: container_registry/container_analysis/create_note.go [hunk 2]
```diff
@@ -20,20 +20,19 @@ import (
...
 func createNote(noteID, projectID string) (*grafeaspb.Note, error) {
 	ctx := context.Background()
-	client, err := containeranalysis.NewGrafeasV1Beta1Client(ctx)
+	client, err := containeranalysis.NewClient(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("NewGrafeasV1Beta1Client: %v", err)
 	}
 	defer client.Close()
-
+
```
Comment (tbpg) [inline container_registry/container_analysis/create_note.go]: Looks like you need to run `gofmt -s` (there's usually a way to have your editor run it for you every time you save). Kokoro failed the `gofmt` check.

If you're running it from command line, you'll want to run `gofmt -w -s .` (write files directly, do extra simplifications, all files in this directory (recursive)).

File: container_registry/container_analysis/poll_discovery_finished.go [hunk 3]
```diff
@@ -20,10 +20,10 @@ import (
 	"context"
 	"fmt"
 	"time"
+	"google.golang.org/api/iterator"
```
Comment (tbpg) [inline container_registry/container_analysis/poll_discovery_finished.go]: Nit: would you mind moving this down one line with the other non-standard library imports?

File: container_registry/container_analysis/create_note.go (hunk 2 above)
Comment (DanSanche) [reply container_registry/container_analysis/create_note.go]: sorry, should have caught that! fixed

**Pull request merged**